modules/claims_api/app/controllers/claims_api/metadata_controller.rb

```diff
@@ -6,14 +6,21 @@ class MetadataController < ::ApplicationController
     skip_after_action :set_csrf_header
     skip_before_action(:authenticate)
 
-    def index
+    def index # rubocop:disable Metrics/MethodLength
```

Comment:
could do something like Jack does here:
https://github.com/department-of-veterans-affairs/vets-api/pull/6843/files#diff-19ec423d0f85fbfb55c61ced75a8e8835420454ab372250da883cfb10e8f057cR94
to get rid of this rubocop disable

Reply:
Here's [my version](https://github.com/department-of-veterans-affairs/vets-api/pull/6845/commits/0c43cd2c7bd1a342c9c93e16c178de37003e9136).